src/frontend/src/components/SearchResults/types.ts

```diff
@@ -0,0 +1,36 @@
+export interface ChargeType {
+  statute: string;
+  expungement_result: any;
+  name: string;
+  disposition: {
+    ruling: string;
+    date: string;
+  };
+}
+
+export interface CaseProp {
```

Comment:
What does Prop mean here? I'm not sure if this is the React "prop" or not. We should keep React-specific terms out of the type structure, since we don't want React implementation details mixed into our types and domain modeling, which should be entirely about the thing itself and not the implementation.

Reply:
I have renamed it to CaseType